src/services/app/user.ts

```diff
@@ -483,6 +487,7 @@ export default class UserService {
             children?: number;
             avatarMediaId?: number;
             pushNotificationToken?: string;
+            email?: string;
```

Comment:
I had tried to use `UserCreationAttributes`,  but there is a conflict of types.
`Argument of type 'UserCreationAttributes' is not assignable to parameter of type 'Partial<User>'.
  Types of property 'trust' are incompatible.
    Type 'AppUserTrustCreation | undefined' is not assignable to type 'AppUserTrust[] | undefined'.
      Type 'AppUserTrustCreation' is missing the following properties from type 'AppUserTrust[]': length, pop, push, concat, and 26 more.`

I could overcome this error of many ways: passing separated to the update function: `this.user.update({ email, active, pushNotification })` ignoring the `trust` attribute, or even forcing the type `any`: `user.update(user as any)`. But I don't know if this ways to fix is better than receive this fields individually in the function

Reply:
Well, I'm against `any` (we wouldn't use typescript) and usually those errors happen when I'm missing something. So I decided to try it myself. It can't be `UserCreationAttributes`, because it is an `update` operation. You need to use `User` instead :wink:

Reply:
done!